src/test/java/com/wikia/webdriver/testcases/Anonymization/AnonymizationTests.java

```diff
@@ -0,0 +1,175 @@
+    String qanon = "QAanon" + timestamp;
+    String passw = "makak";
+    SignUpUser
+        user = new SignUpUser(qanon, credentials.emailAnonymousUserTestWikia, passw,
+                              LocalDate.of(1990, 3, 19));
+    UserRegistration.registerUserEmailConfirmed(user);
+
+    new WikiBasePageObject().loginAs(user.getUsername(), user.getPassword(), wikiURL);
+
+    new ArticleContent().push("asdf", "aaaa");
+
+    new SpecialWikiActivityPageObject().open();
+
+    Assertion.assertStringContains(qanon,qanon);
+  }
+
+
+  @Test
+  public void anonyizationMessageWallTest() {
+
+    Credentials credentials = new Credentials();
+    String timestamp = Long.toString(DateTime.now().getMillis());
+    String qanon = "QAanon" + timestamp;
+    String passw = "makak";
+    SignUpUser
+        user = new SignUpUser(qanon, credentials.emailAnonymousUserTestWikia, passw,
+                              LocalDate.of(1990, 3, 19));
+    UserRegistration.registerUserEmailConfirmed(user);
+
+    new WikiBasePageObject().loginAs(user.getUsername(), user.getPassword(), wikiURL);
+
+    MessageWall wall = new MessageWall().open(user.getUsername());
+    MiniEditorComponentObject mini = wall.triggerMessageArea(true);
+    String message = String.format("%s%s", PageContent.MESSAGE_WALL_MESSAGE_PREFIX, timestamp);
+    String title = String.format("%s%s", PageContent.MESSAGE_WALL_TITLE_PREFIX, timestamp);
+    mini.switchAndWrite(message);
+    wall.setTitle(title);
+    wall.submit();
+    wall.verifyMessageText(title, message, user.getUsername());
+    wall.triggerEditMessageArea();
+    String
+        messageEdit =
+        PageContent.MESSAGE_WALL_MESSAGE_EDIT_PREFIX + Long.toString(DateTime.now().getMillis());
+    mini.switchAndEditMessageWall(messageEdit);
+    wall.submitEdition();
+    wall.verifyMessageEditText(title, messageEdit, user.getUsername());
+
+    try {
+      Thread.sleep(15000);
+    } catch (InterruptedException e) {
+      PageObjectLogging.logError("Interruption during waiting for Message Wall background task",
+                                 e);
+
+
+
+    }
+
+//
+//    wall.open(newName);
+//    String encodedUrl = renameUserPage.encodeToURL(newName);
+//
+//    String expectedWallUrl = String.format("%s/wiki/Message_Wall:%s", urlBuilder.getUrlForWiki(),
+//                                           encodedUrl);
+//
+//    Assertion.assertEquals(driver.getCurrentUrl(), expectedWallUrl);
+//    wall.verifyMessageEditTextRenameDone(title, message, newName);
+//    String
+//        theContent =
+//        EmailUtils.getFirstEmailContent(credentials.emailAnonymousUserTestWikia,
+//                                        credentials.emailAnonymousUserTestWikiaPassword,
+//                                        "Your username change on FANDOM is complete!");
+//  }
+//
+//  @Test
+//  public void anonyizationBlogTest() {
+//
+//    Credentials credentials = new Credentials();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)